### PR TITLE
cla-check: Use has-signed-canonical-cla GitHub Action

### PR DESCRIPTION
**Pull request by MarcusTomlinson**
Use the has-signed-canonical-cla github action instead of our hand-written script.

### CODE REVIEW

**Review by zyga — APPROVED** [review]
Looks good!

**Review by bboozzoo — APPROVED** [review]
Thank you!

File: .github/workflows/cla-check.yaml [hunk 1]
```diff
@@ -6,23 +6,7 @@ on:
 
 jobs:
   cla-check:
-    runs-on: self-hosted
+    runs-on: ubuntu-latest
```
Comment (anonymouse64) [inline .github/workflows/cla-check.yaml:9]: nice! I'm happy we don't have to keep running this locally on self-hosted runners anymore

**Review by anonymouse64 — APPROVED** [review]
thanks so much for all the work to make this happen!

**Pull request merged**